Hyper/Controllers/BoundingShapesController.cs

```diff
@@ -80,8 +81,7 @@ private void Render(BoundingBox box)
         _shader.SetMatrix4("model", localScale * globalScale * rotation * translation);
 
         GL.BindVertexArray(_boxResource.Vaos[0]);
-        GL.DrawElements(PrimitiveType.Triangles, _boxResource.Model.Meshes[0].FaceCount * 3,
-            DrawElementsType.UnsignedInt, 0);
+        GL.DrawArrays(PrimitiveType.Triangles, 0, _boxResource.Model.Meshes[0].FaceCount * 3);
```

Comment:
Can we safely do that?

Reply:
I think we have to do that. From what I understand the drawelements call does not make sensesince we just give it 0 instead of eboId. I think the implementation we have does not make use of indices so the code we had was wrong (worked by chance). I will also remove the ebo creation from modelLoader

Reply:
BTW I think calling DrawElements like we did is undefined behaviour but this is important so if you're not sure about this change let me know